arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/Schedule.kt

```diff
@@ -218,53 +265,81 @@ sealed class Schedule<Input, Output> {
    */
   abstract infix fun <A : Input, B> andThen(other: Schedule<A, B>): Schedule<A, Either<Output, B>>
 
+  @Deprecated(
+    "modifyDelay will be replaced by modify and modifyNanos which use kotlin.time.Duration instead of arrow.fx.coroutines.Duration",
+    ReplaceWith("modifyNanos { output, l -> f(output, l.toLong().nanoseconds).nanoseconds.toDouble() }", "arrow.fx.coroutines.nanoseconds")
```

Comment:
`f(output, l.toLong().nanoseconds).nanoseconds.toDouble()` yep, I definitely prefer the kotlin.Duration version 😄